### PR TITLE
Clean up

### PR DESCRIPTION
**Pull request by CiotkaCierpienia**
example events, readme clean up, correct events on the calendar, dependencies update

### CODE REVIEW

File: src/components/Calendar.js [hunk 1]
```diff
@@ -1,246 +1,251 @@
...
+                                        )}
+                                    >
+                                        {day.date.split('-').pop().replace(/^0/, '')}
+                                    </time>
+                                    {day.events.length > 0 && (
+                                        <ol className="pl-4">
+                                            {day.events.map((event) => (
+                                                <Link to={`/${event.slug}`}>
+                                                    <li key={event.id} className="pb-3">
+                                                        <a href={event.href} className="group flex flex-col">
```
Comment (trzcina) [inline src/components/Calendar.js]: Is this anchor necessary, (or 2 lines above Link)?
What is the difference between event.slug and event.href?
Can we use only Link component here?

**Comment by trzcina** [comment]
I have these non-clickable buttons on mobile:
Flotiq-Gatsby-event-starter (2)

File: src/components/NextMonthsNavigation.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ArrowNarrowLeftIcon, ArrowNarrowRightIcon } from '@heroicons/react/solid';
+
+const NextMonthsNavigation = ({ additionalClass = [], page, numOfPages }) => (
+    <nav className={['max-w-7xl mx-auto grid grid-cols-2 '
+    + 'lg:grid-cols-3 gap-4', ...additionalClass].join(' ')}
+    >
+        {page > 1 && (
+            <div className="col-start-1 col-end-2 bg-white border border-gray">
+                <a
```
Comment (trzcina) [inline src/components/NextMonthsNavigation.js]: Please change A to Link when it is possible

**Comment by ghost** [comment]
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**flotiq-gatsby-event-1**

:tada: Your build was successful! [See the Deploy preview here.](https://flotiqgatsbyevent1-featurecleanup.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/af1e46ca-8601-4912-b25e-aa9d290da183/sites/c7b7b4d4-5d14-410a-ac6c-6c075d803b88/builds/39a88923-9f65-4c05-ba80-7a4fb84bc32d/details)

:clock1: Build time: **1m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 83   |
| Accessibility |   :green_heart: 94   |
| Best Practices |   :green_heart: 100   |
| SEO |   :large_orange_diamond: 85   |

:link: [View full report](https://build-39a88923-9f65-4c05-ba80-7a4fb84bc32d.gtsb.io/reports/lighthouse/index.html)

**Pull request merged**